spec/models/cookbook_spec.rb

```diff
@@ -163,6 +163,17 @@
       expect(cookbook.collaborator_users).to_not include(hank)
     end
 
+    it 'should create a new collaborator record for the previous owner' do
+      hank = create(:user)
+      create(:cookbook_collaborator, resourceable: cookbook, user: hank)
+      expect(cookbook.owner).to eql(jimmy)
```

Comment:
I was keeping with the style of the rest of the file:
https://github.com/chef/supermarket/blob/transfer_ownership/spec/models/cookbook_spec.rb